guides/common/modules/con_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -0,0 +1,7 @@
+[id="configuring-dns-dhcp-and-tftp_{context}"]
+= Configuring DNS, DHCP, and TFTP
+
+With {Project}, you can manage DNS, DHCP, and TFTP centrally within the {Project} environment, or you can manage them independently after disabling their maintenance on {Project}.
+ifndef::foreman-deb,orcharhino[]
+You can also run the services externally, outside of the {Project} environment.
```

Comment:
The sentence above is always present, hence the current wording is fine.

Reply:
I also think `also` is fine, but I like Malhar's suggestion to specify the services again (to avoid potentially making users re-read the list in the first sentence). So I partially applied this suggestion.